generator/src/main/java/org/apache/kafka/message/FieldSpec.java

```diff
@@ -116,6 +119,8 @@ public FieldSpec(@JsonProperty("name") String name,
         }
         this.tag = Optional.ofNullable(tag);
         checkTagInvariants();
+
+        this.zeroCopy = zeroCopy;
```

Comment:
Please add a check that the type is "bytes" if zero copy is enabled.  Throw an exception if it is not

Reply:
fixed.